test/versioned/otel-bridge/span.test.js

```diff
@@ -135,6 +138,80 @@ test('Otel http external span test', (t, end) => {
   })
 })
 
+test('Reconcile Otel undici external span attributes test', (t, end) => {
```

Comment:
i wouldn't call this undici or http. it's just compliant, legacy client http spans, undici being compliant, with that being said you're mixing or including superfluous attrs here. `server.address`  is spec compliant, not `http.host`.  also `url.full` is spec compliant not `http.url`